.github/workflows/continuous-integration-karma-dashboard.yml

```diff
@@ -0,0 +1,42 @@
+name: Dashboard Integration Tests
+
+on:
+  push:
+    branches:
+      - master
+      - release/*
+  pull_request:
+
+jobs:
+  karma:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Read .nvmrc
+        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+        id: nvm
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ steps.nvm.outputs.NVMRC }}
+
+      - name: Setup npm cache
+        uses: pat-s/always-upload-cache@v1.1.4
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-npm-${{ env.cache-name }}-
+            ${{ runner.os }}-npm-
+            ${{ runner.os }}-
+
+      - name: Install dependencies
+        run: npm ci
+        env:
+          CI: true
+
+      - name: Run dashboard integration tests
+        run: npm run test:karma:dashboard -- --headless --viewport=1600:1000
```

Comment:
Just for context why this is also good: we will soon be adding more test versions like this:

```
run: npm run test:karma:dashboard -- --headless --viewport=size2
run: npm run test:karma:dashboard -- --headless --viewport=size3
run: npm run test:karma:dashboard -- --headless --viewport=size2 --browser=firefox
```

etc